### PR TITLE
Update logos

### LINKED ISSUE #298: docs: Add Bolts logo
## 🚀 Feature
Not a strong opinion, but I would like to suggest a change to the logo image located at the top left in https://pytorch-lightning-bolts.readthedocs.io/en/. Currently, both `pytorch-lightning` and `pytorch-lightning-bolts` docs use the same logo image, and I personally think that makes it somewhat confusing. It would be much easier to distinguish them if the `pytorch-lightning-bolts` logo is different from `pytorch-lightning` logo.

### PR DESCRIPTION
**Pull request by edenlightning**
## What does this PR do?

Fixes #298 

## Before submitting
- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together? Otherwise, we ask you to create a separate PR for every change.
- [x] Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests?
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by edenlightning** [comment]
@Borda i can't see my changes reflected in the doc build. any idea?

**Comment by akihironitta** [comment]
@edenlightning Might need to change or somehow override the path here since we use `pt_lightning_sphinx_theme`:
https://github.com/PyTorchLightning/lightning_sphinx_theme/blob/73a7d937ae5423100e1268263f6d1c09edcab585/pt_lightning_sphinx_theme/static/css/theme.css#L9726

**Comment by akihironitta** [comment]
@edenlightning @Borda We currently have no logo on README because https://github.com/PyTorchLightning/pytorch-lightning/pull/4844 removed the old lightning logo image which was also used in Bolts' README.md. Shall we merge this PR so that we can have the new logo image in README.md and continue to apply the new logo to the docs in another PR maybe?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div align="center">
 
-![Logo](https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/master/docs/source/_images/logos/lightning_logo.svg)
+![Logo](https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning/master/docs/source/_images/logos/bolts_logo.png)
```
Comment (Borda) [inline README.md:3]: Bolts shall have own logo in the repo
```suggestion
![Logo](https://raw.githubusercontent.com/PyTorchLightning/pytorch-lightning-bolts/master/docs/source/_images/logos/bolts_logo.png)
```

**Review by akihironitta — APPROVED** [review]
@edenlightning let's merge this PR after applying @Borda's above suggestion :]

**Pull request merged**